AnnoDesigner/Localization/Localization.cs

```diff
@@ -158,6 +158,7 @@ public static void Init(ICommons commons)
                         { "OK" , "OK" },
                         { "Cancel" , "Cancel" },
                         { "SelectAll" , "Select All" },
+                        { "RecentFiles" , "Recent Files" }
```

Comment:
This should be "Recent files" to match the rest of the UI.

Reply:
I have copied it from Visual Studio.
Change it anyway?